
* Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A, Speleman F (2002a) Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol 3:RESEARCH0034
* Vandesompele J, De Paepe A, Speleman F (2002b) Elimination of primer–dimer artifacts and genomic coamplification using a two-step SYBR Green I real-time RT-PCR. Anal Biochem 303:95–98
* Ramakers C, Ruijter JM, Deprez RH, Moorman AF (2003) Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data. Neurosci Lett 13:62–66
* Dheda K, Huggett JF, Bustin SA, Johnson MA, Rook G, Zumla A (2004) Validation of housekeeping genes for normalizing RNA expression in real-time PCR. BioTechniques 37:112–119 . 
* Radonic A, Thulke S, Mackay IM, Landt O, Siegert W, Nitsche A (2004) Guideline to reference gene selection for quantitative real-time PCR. Biochem Biophys Res Commun 23:856–862
* Dheda K, Huggett JF, Chang JS, Kim LU, Bustin SA, Johnson MA, Rook GAW, Zumla A (2005) The implications of using an inappropriate reference gene for real-time reverse transcription PCR data normalization. Anal Biochem 344:141–143 . 
* Hugget J, Dheda K, Bustin S, Zumla A (2005) Real-time RT-PCR normalisation; strategies and considerations. Genes Immun 6:279–284 
* Perez-Novo CA, Claeys C, Speleman F, Cauwenberge PV, Bachert C, Vandesompele J (2005) Impact of RNA quality on reference gene expression stability. BioTechniques 39:52–56
* Fleige S, Walf V, Huch S, Prgomet C, Sehm J, Pfaffl MW (2006) Comparison of relative mRNA quantification models and the impact of RNA integrity in quantitative real-time RT- PCR. Biotechnol Lett 28:1601–1613
* Fleige S, Pfaffl MW (2006) RNA integrity and the effect on the real-time qRT-PCR performance. Mol Aspects Med 27:126–139
* Rieu, I., and Powers, S. J. (2009). Real-time quantitative RT-PCR: design, calculations, and statistics. Plant Cell 21, 1031–1033. doi: 10.1105/tpc.109.066001
* Derveaux S, Vandesompele J, Hellemans J (2010) How to do successful gene expression analysis using real-time PCR. Methods 50: 227–230.


the qBase software (Hellemans et al. 2007) 
  
This is a summary of major steps steps required throughout the entire workflow that may influence the accuracy and reliability 
of results.

Bustin SA, Benes V, Garson JA, Hellemans J, Huguett J, Kubista M, Mueller R et al (2009) The MIQE Guidelines: minimum information for publication of quantitative real-time PCR experiments. Clin Chem 55:611–622


# Plants
Undoubtedly, the plant research community benefits from biomedical analyses, and the introduction of different technologies 
usually occurs after they have been implemented in mammalian or bacterial research. These problems have been known and 
publicized for quite a long time, and the plant scientific community has gradually recognized the need for more robust 
standards

In the field of plants, Gutierrez et al. (2008b) elegantly demonstrated recently that a reference gene with stable expression in one organism may exhibit a different expression pattern in another organism; even when studying the same organism, no reference gene could be considered as a universal reference, signifying the importance of studies aimed at the identification and/or validation of the expression stability of certain reference genes. In a high-impact study performed by Czechowski et al. (2005), it was shown that the identification of novel genes that had not previously been considered as internal controls could outperform traditional references in terms of expression stability.
The number of studies aimed at validating the presumed stability in the expression of certain reference genes studies in human and animal research contrasts with the very little attention that the validation and/or identification of reference genes has received in the plant sciences (Gutierrez et al. 2008a)
the importance of systematic validation has not been integrated in qPCR analysis (Guénin et al. 2009). As a consequence, genes currently used as references for qPCR analysis in plants are almost exclusively putative reference genes (Gutierrez et al. 2008b). 



#qBASE
CV 
The maximisation method as PCR run set-up was chosen because it has the important advantage of not suffering from technical (run-to-run) variation between the samples (Hellemans et al. 2007).


* Udvardi MK, Czechowski T, Scheible W-R (2008) Eleven golden rules of quantitative RT-PCR. Plant Cell 20:1736–1737
* Gutierrez L, Mauriat M, Guénin S, Pelloux J, Lefebvre JF, Louvet R, Rusterucci C et al (2008a) The lack of a systematic validation of reference genes: a serious pitfall undervalued in reverse transcription-polymerase chain reaction (RT-PCR) analysis in plants. Plant Biotechnol J 6:609–618
* Guénin S, Mauriat M, Pelloux J, Van Wuytswinkel O, Bellini C, Gutierrez L (2009) Normalization of qRT-PCR data: the necessity of adopting a systematic, experimental conditions-specific, validation of references. J Exp Bot 60:487–493
* Gutierrez L, Mauriat M, Pelloux J, Bellini C, Van Wuytswinkel O (2008b) Towards a systematic validation of references in real-time RT-PCR. Plant Cell 20:1734–1735
* Graeber, K., Linkies, A., Wood, A. T. A., and Leubner-Metzger, G. (2011). A guideline to family-wide comparative state-of-the-art quantitative RT-PCR analysis exemplified with a Brassicaceae cross-species seed germination case study. Plant Cell 23, 2045–2063. doi: 10.1105/tpc.111.084103
* De Keyser, E., Desmet, L., Van Bockstaele, E., and De Riek, J. (2013). How to perform RT-qPCR accurately in plant species? A case study on flower colour gene expression in an azalea (Rhododendron simsii hybrids) mapping population. BMC Mol. Biol. 14:13. doi: 10.1186/1471-2199-14-13

